WMF Framework/SharedContainerCache.swift

```diff
@@ -25,20 +28,79 @@ public final class SharedContainerCache<T: Codable> {
         return cacheDirectoryContainerURL.appendingPathComponent(pathComponent.rawValue).appendingPathExtension("json")
```

Comment:
This could be reworked to:

```
private var cacheDataFileURL: URL {
    let baseURL = subdirectoryURL ?? cacheDirectoryContainerURL
    return baseURL.appendingPathComponent(fileName).appendingPathExtension("json")
}
```